messaging/methods/messages/createMessage.md

```diff
@@ -324,6 +403,60 @@ Content-Type: application/json; charset=utf-8
   "segmentCount" : 1
 }
 ```
+
+{% sample lang="csharp" %}
+
+```csharp
+MessageRequest messageRequest = new MessageRequest();
+messageRequest.ApplicationId = MSG_APPLICATION_ID;
+messageRequest.To = new List<string> { "+12345678902" };
+messageRequest.From = "+12345678901";
+messageRequest.Text = "Hey, check this out!";
+messageRequest.Tag = "test message";
+messageRequest.Media = new List<string> { "https://s3.amazonaws.com/bw-v2-api/demo.jpg", "https://s3.amazonaws.com/bw-v2-api/demo2.jpg"  };
+
+var response = msgClient.CreateMessage(MSG_ACCOUNT_ID, messageRequest);
```

Comment:
Do we want to demonstrate error handling with C#? Or remove error handling from ruby/python?

Reply:
Are you talking about try catch blocks?

Reply:
For C# and Java since the are compiled IDEs will enforce the try catch or throw.  Normally I would leave it up to the developer to decide where to put it.

Reply:
Gotcha, I'm fine leaving it off then if you're fine with that

Reply:
I'd rather add error handling to C#